### PR TITLE
workflow settings changelog

### PR DESCRIPTION
**Pull request by m-abboud**

### CODE REVIEW

File: fern/docs/content/change-log/2024-02.mdx [hunk 1]
```diff
@@ -2,6 +2,17 @@
 title: Changelog | February, 2024
 ---
 
+
+## Workflow Builder UI Settings
+
+_February 21st, 2024_
+
+Have you ever wanted to pan around workflows using the W-A-S-D keys? Are you tired of the mini map taking up a small portion of your screen?
```
Comment (dvargas92495) [inline fern/docs/content/change-log/2024-02.mdx:10]: [nit] I would rephrase the second one so that it's more affirmative instead of antagonistic towards something we intend to keep as a feature. Something like
- `Looking for more control over your screen real estate?`
- (or a highlight of one of the other settings)

**Review by dvargas92495 — APPROVED** [review]

**Review by noanflaherty — COMMENTED** [review]
Love this feature!

for consistency, I’d prefer that we avoid the use of first person and follow the style of the precedents set.
